snowflake/cmd/receiveSqsMessage.go

```diff
@@ -97,7 +99,7 @@ func receiveSQSMessage(ctx context.Context, sqsClient sqsclient.Interface, queue
 		return nil
 	}
 	message := output.Messages[0]
-	fmt.Println(*message.Body)
+	println(*message.Body)
```

Comment:
rather than this change, I'd prefer to add an extra parameter to the `receiveSQSMessage` function (of type `io.Writer`). It's cleaner and a very common solution.